packs/actions/high-jump.json

```diff
@@ -11,7 +11,7 @@
         },
         "category": "interaction",
         "description": {
-            "value": "<p>You Stride, then attempt a DC 30 <span data-pf2-action=\"highJump\" data-pf2-glyph=\"D\">Athletics</span> check to jump vertically. If you didn't Stride at least 10 feet, you automatically fail. This DC might be increased or decreased due to the situation, as determined by the GM.</p>\n<hr />\n<p><strong>Critical Success</strong> You @UUID[Compendium.pf2e.actionspf2e.Item.Leap] up to 8 feet vertically and 10 feet horizontally.</p>\n<p><strong>Success</strong> You Leap up to 5 feet vertically and 5 feet horizontally.</p>\n<p><strong>Failure</strong> You Leap normally.</p>\n<p><strong>Critical Failure</strong> You fall @UUID[Compendium.pf2e.conditionitems.Item.Prone] in your space.</p>"
+            "value": "<p>You Stride, then attempt a <span data-pf2-action=\"highJump\" data-pf2-glyph=\"D\" data-pf2-dc=\"30\">Athletics</span> check to jump vertically. If you didn't Stride at least 10 feet, you automatically fail. This DC might be increased or decreased due to the situation, as determined by the GM.</p>\n<hr />\n<p><strong>Critical Success</strong> You @UUID[Compendium.pf2e.actionspf2e.Item.Leap] up to 8 feet vertically and 10 feet horizontally.</p>\n<p><strong>Success</strong> You Leap up to 5 feet vertically and 5 feet horizontally.</p>\n<p><strong>Failure</strong> You Leap normally.</p>\n<p><strong>Critical Failure</strong> You fall @UUID[Compendium.pf2e.conditionitems.Item.Prone] in your space.</p>"
```

Comment:
It might not show the DC inline, so if it doesn't you might still want the DC listed in the description.